frontend/scss/components/templates/policies.scss

```diff
@@ -19,7 +19,7 @@
 @import '../templates/_default.scss';
 
 
-.#{template('code-of-conduct')} {
+.#{template('policies')} {
```

Comment:
`policies` sounds a bit specific. What do you think of `meta`?

Reply:
sounds good to me 👍